### ISSUE TITLE
About FID result

### ISSUE THREAD

**Issue by mzsqw** [cure-lab/MagicDrive#88]
Hi, I was wondering how many epochs you trained on your released checkpoint? I noticed that the fid of the pre-generated samples you provided for download in the "To test fid" section is 14.46065995481922, while the FID I got after training 100 epochs according to the default setting is 13.288919496375172.
![image](https://github.com/user-attachments/assets/26b09978-838e-47e6-af09-4d22be4e88ac)
Perhaps training more epochs is to improve segmentation and detection metrics?

**Comment by flymin** [comment]
Yes. You may also want to evaluate the control accuracy to check the overall performance.
